packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,71 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - Data Science Institute</title>
+
+    <environment include="Development">

```

Comment:
Is this a development-only file? I thought @DavidAkroyd did have a point re:Development raised earlier? Can we do an overall scan and get rid of this?

Reply:
no this is a production file. The line you are considering is the Razor equivilent of #if debug

Reply:
hmm, :). They should have used a better name, :). But don't they let you give your own names for the ASPNETCORE_ENVIRONMENT variable? This looks a bit ugly as-is.